jobs/blockhead/templates/eth.json.erb

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "eth",
+    "description": "Ethereum Geth Node",
+    "tags": [
+        "eth",
+        "geth"
+    ],
+    "display_name": "Geth 1.8",
+    "plans": [
+        {
+            "name": "geth-dev",
+            "free": true,
+            "image": "nimak/eth-dev:latest",
+            "description": "Free Service that deploys a geth node in development mode",
+            "ports": ["8545"]
+        }
+    ]
+}
```

Comment:
embedding the service config into the job template is a bit counter intuitive. This basically means that for every service that a user wants to add to the release, they need to also modify the spec and rename the file to be a `json.erb` file.

Can we instead fix this during packaging, for example by copying everything under `src/github.com/cloudfoundry-incubator/blockhead/services` (or alternatively a dedicated `services/` directory in the release) into `${BOSH_INSTALL_TARGET}/services/` and referencing that in the bpm script.

Reply:
That seems a little weird to me. I didn't think services should be part of the blockhead source code package. Other thing I don't like about that approach is that we are tying the release to sample files in blockhead. I would think the bosh release should be where the source of truth of services not the blockhead source code.

Reply:
I agree. That's why I suggested maybe we can have a `services/` directory in the bosh release and we services files from there into the deployment. The directory doesn't need to have anything in it to begin with, but if created, we can copy arbitrary set of services files through to the deployment. Let's discuss it further in the meeting.